src/BuildScriptGenerator/Hugo/HugoPlatformInstaller.cs

```diff
@@ -32,6 +32,10 @@ public virtual string GetInstallerScriptSnippet(string version)
             var snippet = new StringBuilder();
             snippet
                 .AppendLine()
+                .AppendLine("IMAGES_DIR=\"/opt/tmp/images\"")
```

Comment:
This change ensures that we don't need to pre-bake Hugo for all images and only install it at run-time for the CLI image, correct?

Reply:
Added this line here to run installHugo.sh script here _only_ for jamstack image so hugo and it's dependencies (golang) are installed dynamically.

For the cli image, however, [this](https://github.com/microsoft/Oryx/blob/main/src/BuildScriptGenerator/Node/NodePlatformInstaller.cs#L43) line runs the installHugo.sh script.